src/BlockchainTestsFiller/ValidBlocks/bcEIP1559/intrinsicFiller.yml

```diff
@@ -0,0 +1,2327 @@
+    0xFCb7d712eFc408D2b78Cd698f2025D1A85D83a84 :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x73ac178a6eb7f087e8bfa080a72680ee300f1a62ef86a006c6c423093926a96d 
+    0x3B49C58fEca066328089cFFeBAF751ee2Afe71DE :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x2bc71c7ad3b4903dc71fa186489becda207981b8449ed9189179a6bea59b502e 
+    0x03D6DBaf9684C9bbA818A9bE1F66300e8DeEc8fB :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x9f4f430d8f79df6d6c1d426fa08f1f59e79004e5c6c6236f7135661e54a3a74a 
+    0x0F47FBBC2aB600cff2D8da94Dd53c8De11fA8EC2 :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x195eb7404510ccd7b16f1ec503f56dd93d476310ac5c93606d307085059d0da5 
+    0xF5426f5D6F46Eb6a861adEdD37da9F3bd4EeB05B :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x64a10835f413d6cdac0c8b59cec615204315cf4d3c7dd2ccb4266d85bd033865 
+    0x7A0Afb8c22D099e8d9353207c7053174b71F9fF0 :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0xd6e5e788dba82de1e3fe79186f227a5a8fa88276c4b48a243eb3e99efb951927 
+    0xA1e2E8702c87DfB9B3882b319eEEf6cfF9E289Ad :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x1aa5f953967356752f108fdd6a296e94720c512883b334e7fe70dac58a7ca550 
+    0x9BbA6A5A1289EB02Df3Ff4bB66d526d1f33ae92c :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0xec278e5d731a2516d0145a614f8ee246b66ea0d0c7ea713cb9cbe69deefcb7f2 
+    0xd58cB804098D02aa6EF99982D04BAF36D784fA7F :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x2d96fb8c2c090167f8e361ff841d9b968a9704430351ded53888ee1ea9ff2a7f 
+    0xabb152795d0a1c12ced3b6c67839F8dfC1b69C98 :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0xef040539fae730dad35ce293cf4960102acc079d98564963bfa7ea3e401a2f63 
+    0xE65253613f215241c99567F89Cc814567a481116 :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x57fb1a068a0c892cdf9b69e1bde47f455159c0f9d83d337435a557bae3d4f724 
+    0xC13928e81b595D2256c9753fC7989Ac902d0CEE7 :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0xd4de9ad011f36f2fbadca90cc991183780ab4b9f1c2ecffca62cf8fafdab0a32 
+    0x83c85c0dd6F552a61743FA28F94c92C34Ba12F8a :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+# 0x2557957cae9a2df65d00fb9be2fb150d71fa3d148dacf9a44ffe60baa3a9ff92 
+    0xBc335622F9f0316D2435507aF3F543424906B7b5 :
+      balance: 100000000000
+      nonce: 1
+      code: 0x
+      storage: {}
+
+
+
+  blocks:
+  
+  - transactions:
+    # Verify gas cost
+    - data: ''
+      accessList: []
+      gasLimit: 500000	
+      maxPriorityFeePerGas: 0x00
```

Comment:
one more check to see that maxPriority (tip) does not affect formula calculation

Reply:
The formula I use to get the formula result depends on having the same gasprice. So it's simplest to create another test file with the same tip for every transaction.

Done.